src/components/external/navbar.vue

```diff
@@ -63,39 +70,39 @@
           close-icon
           scheme="feedback-red"
           modal-icon="logout-1-1"
-          :description="getTranslation('NAVBAR.LOGOUT_MESSAGE')"
-          :text="getTranslation('NAVBAR.LOGOUT')"
+          :description="$t('NAVBAR.LOGOUT_MESSAGE')"
+          :text="$t('NAVBAR.LOGOUT')"
           @close="logoutModalOpen = false">
           <unnnic-button
             slot="options"
             @click="logoutModalOpen = false"
             type="terciary">
-              {{ getTranslation('NAVBAR.CANCEL') }}
+              {{ $t('NAVBAR.CANCEL') }}
             </unnnic-button>
           <unnnic-button
             class="weni-button-danger"
             slot="options"
             @click="logout()">
-              {{ getTranslation('NAVBAR.LOGOUT', language) }}
+              {{ $t('NAVBAR.LOGOUT', language) }}
             </unnnic-button>
         </unnnic-modal>
     </div>    
 </template>
 
 <script>
-import { unnnicDropdown, unnnicButton, unnnicModal, unnnicAutocomplete } from '@weni/unnnic-system';
+import { unnnicButton, unnnicModal, unnnicAutocomplete, unnnicDropdown } from '@weni/unnnic-system';
```

Comment:
perhaps you can import unnnic in your app.vue like that import unnic from '@weni/unnnic-system';
and unnnic components will be available on every views/component no need to import in each component, could u try it ?

Reply:
yes ;) you're right. We will need to remove it from other pages. I will plan this update, thanks.

Reply:
great!! :)